test/e2e/websocket_test.go

```diff
@@ -181,7 +181,7 @@ func TestWebSocketViaActivator(t *testing.T) {
 	}
 }
 
-func TestWebSocketBlueGreenRoute(t *testing.T) {
+func TestWebSocketBlueGreen(t *testing.T) {
```

Comment:
This test name creates `green-web-socket-blue-green-route-fswjwwfo.serving-tests.example.com` by adding tag name as:
```
    websocket.go:65: Connecting using websocket: url=wss://a0a9239245fa34063ba2446f5ef425cc-273904986.ap-southeast-1.elb.amazonaws.com:443/, host=green-web-socket-blue-green-route-fswjwwfo.serving-tests.example.com
```

And it hits net-certmanager's issue https://github.com/knative-sandbox/net-certmanager/issues/214